src/platform_impl/linux/wayland/scale.rs

```diff
@@ -0,0 +1,52 @@
+use std::ops::Mul;
+
+use dpi::{LogicalSize, PhysicalSize};
+
+/// A wp-fractional-scale scale.
+///
+/// This type implements the `physical_size = round_half_up(logical_size * scale)`
+/// operation with infinite precision as required by the protocol.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct Scale {
+    scale: u64,
+}
+
+const BASE: u32 = 120;
+const BASE_U64: u64 = BASE as u64;
+const BASE_F64: f64 = BASE as f64;
```

Comment:
it's used exactly in one place, so you can inline and just call it `DENOMINATOR: u64 = 120`.